### PR TITLE
[DOCS] Add docs around Snowflake FDS

### PR DESCRIPTION
**Pull request by cdkini**
Add section to existing guide to reflect new Snowflake FDS

- [x] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Code is linted - run `invoke lint` (uses `black` + `ruff`)
- [x] Appropriate tests and docs have been updated

For more details, see our [Contribution Checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist), [Coding style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style), and [Documentation style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/docs_style).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md
Comment (cdkini) [inline docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md:2]: Changed `Source Data` to `source data` throughout doc

File: docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md
Comment (cdkini) [inline docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md:60]: Changed `PostGreSQL` to `PostgreSQL` throughout doc

File: docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md
Comment (cdkini) [inline docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md:125]: Wasn't rendering properly due to lack of space

File: docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md
Comment (cdkini) [inline docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md:208]: Was referencing Postgres but in the wrong section

File: docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md]: You fixed the reference to PostgreSQL that should have been SQLite in the line above; can you change it in this one as well?

```suggestion
The following code is an example of a SQLite connection string format:
```

File: docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md:311]: This info block isn't rendering correctly (possibly because of the use of the backticks in the header or a missing newline between the last line of text and the final `:::`.  I'm not sure how strict markdown is with permitted formatting here).  You should adjust this and double check the rendering before merging.

**Review by Rachel-Reverie — APPROVED** [review]
Two minor edits.  I'm approving so that you aren't blocked after corrections are made.

**Pull request merged**
